corehq/apps/hqadmin/views.py

```diff
@@ -1004,6 +1004,8 @@ def _gir_csv_response(month, year):
     query_month = "{year}-{month}-01".format(year=year, month=month)
     prev_month = "{year}-{month}-01".format(year=year, month=month - 1)
     two_ago = "{year}-{month}-01".format(year=year, month=month - 2)
+    if not GIRRow.objects.filter(month=query_month).exists():
+        return HttpResponse('Sorry, that month is not yet available')
```

Comment:
It doesnt use any template, but since only dimagi users can see this (and really only jeremy uses it), that seems fine.